### PR TITLE
Adds the uniform vendor

### PR DESCRIPTION
**Pull request by Cupax3**
Deadmon convinced me to spitecode it back in.
This is on you, deadmon.

## Description of changes
Ports the uniform vendor from Bay.

## Why and what will this PR improve
This is spitecoded & spiteported because I want to be fabulous while playing spess, thank you.
<details>
<summary>Screenshots</summary>

image
image
image

</details>

## Authorship
NullSnaphot, SamSamsonthe3rd, Daniel Thompson, Spookerton, comma, afterthought, PsiOmegaDelta, MistakeNot4892, HarpyEagle, sabiram, Asanadas, mikomyazaki, CakeQ, AtomicWorm, SkyMarshal
Listed in order of git blame for uniform_vendor.dm

## Changelog

:cl: Many Bay & Nebula coders from 5 years ago
rcsadd: Makes the uniform vendor vendor uniforms again. No more lost IDs, no more (un)fabulous fatigue fest.
/:cl:

### CODE REVIEW

**Comment by Mazianni** [comment]
thank you for doing the thing i was procrastinating on doing

File: maps/torch/items/uniform_vendor.dm [hunk 1]
```diff
@@ -36,4 +37,171 @@
...
+					if(piece in selected_outfit)
+						dat += "<span class='linkOn'>[sanitize(initial(C.name))]</span><a href='byond://?src=\ref[src];rem=\ref[piece]'>X</a>"
+					else if (can_issue(C))
+						dat += "<a href='byond://?src=\ref[src];add=\ref[piece]'>[sanitize(initial(C.name))]</a>"
+					else
+						dat += "[sanitize(initial(C.name))] (ISSUED)"
+			dat += "<hr>"
+		dat += "<a href='byond://?src=\ref[src];vend=[1]'>Dispense</a>"
+	dat = jointext(dat,"<br>")
+	var/datum/browser/popup = new(user, "Uniform Dispenser", "Uniform Dispenser", 300, 700, src)
```
Comment (Mazianni) [inline maps/torch/items/uniform_vendor.dm:69]: could i ask you to convert this to nanoUI

Comment (Cupax3) [reply maps/torch/items/uniform_vendor.dm:69]: I have literally never worked with nanoui

Comment (Mazianni) [reply maps/torch/items/uniform_vendor.dm:69]: alright nvm

**Pull request merged**